crates/oxc_transformer/src/helpers/ast_helper.rs

```diff
@@ -0,0 +1,43 @@
+#[macro_export]
+macro_rules! ast_helper {
```

Comment:
I like this macro. That being said, `helpers` sets off some red flags for me. If we want to support AST-related macros, we should use domain-specific folders and macro names.

Reply:
Sorry to see it after closing :-), but actually the `helpers` does not apply directly in the AST. [Here](https://github.com/oxc-project/oxc/issues/4753#issuecomment-2354216972) I think it should be a better solution, and I am trying to work on it.